src/style/colours.less

```diff
@@ -9,8 +9,12 @@
 @pale-blue: #81bee8;
 @paler-blue: #add4f0;
 @palest-blue: #d9ebf8;
-@green: #00A55E;
-// + shades by 
+@green: #00A55E; 
+//colours from Brian
+@dark-turquoise: #39917F;
+@white: #FFFFFF;
```

Comment:
white is a built-in, e.g. `color:white` works, and we don't plan to edit it - so no need to define it